networks/local/blockchain-local/Dockerfile

```diff
@@ -36,11 +35,70 @@ WORKDIR /provenance
 EXPOSE 26656 26657 1317 9090
 
 # Source binaries from the build above
-COPY --from=build /go/src/github.com/provenance-io/provenance/vendor/github.com/CosmWasm/wasmvm/api/libwasmvm.so /lib/x86_64-linux-gnu/libwasmvm.so
-COPY --from=build /go/bin/provenanced /usr/bin/provenanced
+COPY --from=build-x86_64 /go/src/github.com/provenance-io/provenance/vendor/github.com/CosmWasm/wasmvm/api/libwasmvm.x86_64.so /lib/x86_64-linux-gnu/
+COPY --from=build-x86_64 /go/bin/provenanced /usr/bin/provenanced
+
+COPY networks/local/blockchain-local/entrypoint.sh /usr/bin/entrypoint.sh
+
+STOPSIGNAL SIGTERM
 
 ENTRYPOINT ["/usr/bin/entrypoint.sh"]
 CMD ["start"]
-STOPSIGNAL SIGTERM
+
+
+
+
+
+
+## Build provenance for ARM
+FROM golang:1.17-buster as build-arm64
+WORKDIR /go/src/github.com/provenance-io/provenance
+ENV GOPRIVATE=github.com/provenance-io
+RUN apt-get update && apt-get upgrade -y && apt-get install -y libleveldb-dev
+
+COPY client/ ./client/
+COPY app/ ./app/
+COPY go.* ./
+COPY cmd/ ./cmd/
+COPY internal/ ./internal/
+COPY x/ ./x/
+COPY vendor/ ./vendor/
+
+# Build and install librdkafka
+RUN git clone --depth 1 --branch v1.8.2 https://github.com/edenhill/librdkafka.git && cd librdkafka && ./configure --enable-static && make && make install
+
+# Build the binaries
+RUN go build -mod vendor -tags=cleveldb,dynamic -ldflags '-w -s -X github.com/cosmos/cosmos-sdk/version.Name=Provenance-Blockchain' -o /go/bin/ ./cmd/...
+
+## Run provenance for ARM
+FROM debian:buster-slim as provenance-arm64
+ENV LOCALNET=1
+
+RUN apt-get update && \
+    apt-get upgrade -y && \
+    apt-get install -y curl jq libleveldb-dev && \
+    apt-get clean && \
+    rm -rf /var/lib/apt/lists/
+
+# Setup librdkafka
+RUN apt-get update
+RUN apt-get install -y pkg-config
+COPY --from=build-arm64 /usr/local/include/librdkafka /usr/local/include/
+COPY --from=build-arm64 /usr/local/lib/librdkafka* /usr/local/lib/
+
+# Requires a mounted volume with the config in it.
+VOLUME [ "/provenance" ]
+WORKDIR /provenance
+EXPOSE 26656 26657 1317 9090
+
+# Source binaries from the build above
+COPY --from=build-arm64 /go/src/github.com/provenance-io/provenance/vendor/github.com/CosmWasm/wasmvm/api/libwasmvm.aarch64.so /lib/aarch64-linux-gnu/
```

Comment:
Target path is `/lib`? That's odd.

Reply:
Agreed, I was following the convention of the x86 lib. I'll take a look at both of them, and I doubt they have some type of requirement for this path.